homeassistant/components/abode/light.py

```diff
@@ -102,10 +103,26 @@ def hs_color(self) -> tuple[float, float] | None:
         return _hs
 
     @property
-    def supported_features(self) -> int:
-        """Flag supported features."""
+    def color_mode(self) -> str | None:
+        """Return the color mode of the light."""
+        if self._device.is_dimmable and self._device.is_color_capable:
+            if self.hs_color is not None:
+                return COLOR_MODE_HS
+            return COLOR_MODE_COLOR_TEMP
```

Comment:
Is this the correct way to determine if the light is in color mode or in white mode?